firmware/crates/vqf/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "vqf"
+version = "0.1.0"
```

set this version to 0.0.0 for now, until we black box test against vqf-cxx. *then* we can publish to crates.io
```suggestion
version = "0.0.0"
```